cider/function/scalar/RuntimeFunctions.cpp

```diff
@@ -394,6 +394,23 @@ extern "C" ALWAYS_INLINE int8_t bit_is_set(const int64_t bitset,
              : 0;
 }
 
+extern "C" ALWAYS_INLINE int8_t bit_is_set_cider(const int64_t bitset,
```

Comment:
possible to pass a pointer instead?

Reply:
Do you mean the `bitset`? it's the address of the bitmap and the bit check is based on interpreting it, as `reinterpret_cast<const int8_t*>(bitset))[bitmap_idx >> 3` so I think no need to pass a pointer.

Reply:
as a function, it would be better to use int8_t* as input parameter which is more friendly to its user. There may be some limitation in code gen stage (like you cannnot pass a pointer directly when call this external function), so you do like this, right?

Reply:
Understand your point. But for this function's use scenario, it's more straight to pass this `int64_t` address directly because the pattern is fixed, as well as the `int64_t val, val_min, val_max`.

Reply:
Please refer `check_bit_vector_set` `check_bit_vector_clear` in RuntimeFunction.cpp

Reply:
The bit check logic inside function `check_bit_vector_set ` is based on interpreting the `bit_vector` as `uint8_t` and that's make it consistent. In my user scenario, the bit set logic do 'reinterpret_cast<const int8_t*>' inside the function but for the function parameters, it uses int64_t. I think there is no big difference.

Reply:
@jikunshang Thanks for your advice. Per offline discussion, we keep function like this for the time being and will consider reimplement it when switch to more efficient Bitmap late.